samples/MvcSubAreaSample.Web/Areas/Restaurant/Areas/Hours/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace MvcSubAreaSample.Web.Areas.Restaurant.Hours.Controllers
```

Comment:
I'm torn between like you said, avoiding confusion and the fact that putting these in at the base namespace is never how someone would/should do it in a real site. @rynowak thoughts?